nixos/modules/services/monitoring/sysstat.nix

```diff
@@ -0,0 +1,91 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let
+  cfg = config.services.sysstat;
+in {
+  options = {
+    services.sysstat = {
+      enable = mkOption {
+        type = types.bool;
+        default = false;
+        description = ''
+          Whether to enable sar system activity collection.
+        '';
+      };
+
+      package = mkOption {
```

Comment:
I always am frustrated when I look at a service and I can't do it, so I default to having this configurable. I could remove it, though.